examples/chapter1/CMakeLists.txt

```diff
@@ -49,9 +49,31 @@ TARGET_LINK_LIBRARIES(${PROJ_NAME}_3
   ${KLEPSYDRA_CORE_LIBRARIES}
   )
 
-TARGET_INCLUDE_DIRECTORIES(${PROJ_NAME}_3 
+TARGET_INCLUDE_DIRECTORIES(${PROJ_NAME}_3
+  PRIVATE
+  ${CMAKE_CURRENT_SOURCE_DIR}/include
+  )
+
+ADD_EXECUTABLE(${PROJ_NAME}_3-jlbarros ${${PROJ_NAME}_HEADERS} src/example3-jlbarros.cpp )
+TARGET_LINK_LIBRARIES(${PROJ_NAME}_3-jlbarros
```

Comment:
NEVER call a project on personal name.
Please refactor this into a self-explanatory name

Reply:
agree, this example is just a sandbox for your review, no intention to be added to the main branch. It will be renamed as soon as it does something useful.

Reply:
moved to [KCE-60](https://klepsydra.atlassian.net/browse/KCE-60), this branch will be removed.